ntoskrnl/io/iomgr/error.c

```diff
@@ -484,15 +484,35 @@ IopLogWorker(IN PVOID Parameter)
     ExFreePoolWithTag(Message, TAG_IO);
 }
 
-VOID
+/* See also ntoskrnl/io/iomgr/irp.c!IopCompleteRequest */
+static VOID
```

Comment:
Aren't we usually do
```suggestion
static
VOID
```
?

Reply:
Dunno; the former takes slightly less place ^^

Reply:
Why not `static VOID NTAPI` then?)